### ISSUE TITLE
Public Sans - Feature: License under Public Domain

### ISSUE THREAD

**Issue by BoQsc** [uswds/public-sans#332]
### Is your feature request related to a problem? Please describe.

I'm frustrated with attribution and licenses. 
I would like to simply use the font instead of care about if I have included or attributed.
It can get really messy with licenses and attributions in the long term if a project includes many different works that strictly require attribution.

### Describe the solution you'd like

I would like to know if this font will be licensed for Public Domain use. 
Specifically: https://creativecommons.org/public-domain/cc0/

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://designsystem.digital.gov/about/community/#community-conduct).
- [x] I checked the [current issues](https://github.com/uswds/public-sans/issues) for duplicate feature requests.

**Comment by otherjoel** [comment]
Public Sans was created by employees of the US government; therefore by US law it cannot be subject to copyright, and is public domain (see #30). You may as well close this issue.

**Comment by moyogo** [comment]
@otherjoel only the modification made by US government employees.

> This typeface is based off of https://github.com/impallari/Libre-Franklin. Public Sans is a Latin-only font.

@BoQsc Libre Franklin would have to be public domain first.

**Comment by otherjoel** [comment]
> only the modifications made by US government employees are in the public domain.

They aren't distributing the modifications. They are distributing a complete derivative work (a set of complete fonts). If they were only distributing a patch that people could apply to Libre Franklin then there might be an argument there.

**Comment by davelab6** [comment]
> > only the modifications made by US government employees are in the public domain.
> 
> They aren't distributing the modifications. They are distributing a complete derivative work (a set of complete fonts). If they were only distributing a patch that people could apply to Libre Franklin then there might be an argument there.

By this logic, then, it's ofl.

@BoQsc what's your concern about the ofl?

**Comment by otherjoel** [comment]
> By this logic, then, it's ofl.

To place the derivative work under the OFL, the GSA needs to be holding a copyright to its contributions, which it does not (and cannot) have. If it cannot hold copyright, it cannot place the work under the terms of a license.

**Comment by moyogo** [comment]
@otherjoel If you publish a book or a song under a permissive license that must remain the same in derivative works and the GSA changes some words in 1% of the sentences, does the license change? What if the GSA changes somes words in 50% of the sentences? What if the GSA changes some words in 99% of the sentences, or in 100% of the sentences? Or what if the GSA translates your work into a different language altogether? Whatever work the GSA has done may be considered to be under public domain, what hasn’t changed or what it started from remains under the permissive but remaining license. But because the license says any derivative must have the same license, the derivative work of the GSA has the same license.

**Comment by davelab6** [comment]
I agree with Denis. To place the derivative work under the OFL, the GSA
does not need to be holding any copyright to its contributions, which I
agree it does not (and cannot) have; since its contributions have no
copyrights and thus no license, their contributions are 'transparent', and
don't effect the underlying/original license situation at all.

While it cannot hold copyright, and can not place a new work under the
terms of a license, the existing work is already placed under the terms of
a licence, and that remains in place after the public-domain contributions
are added.

**Comment by otherjoel** [comment]
@moyogo 

> the GSA changes some words in 1% of the sentences

If the GSA distributes the book/song in its entirety with their changes, they have created a derivative work.

> …does the license change?

The license…of what? 

Of the original? No, of course not.

Of my derivative work? That would be a wrong question. My derivative work has no license until I decide to license it. I am enjoined to use the parent work under the terms of *its* license; but it is up to me to decide on licensing terms for users of my derivative work.

Derivative works do not automatically “inherit” license terms from the works they derive from. If the parent work is distributed under the terms of a license that makes stipulations about how I distribute derivative works, I am required to follow those stipulations — but they don’t happen “by default”.

Put it another way — the OFL doesn't say “your derivative fonts are automatically OFL licensed too.” Rather it says “modified fonts must be distributed entirely under this license”, enjoining authors of derivative fonts to take action, i.e. place their modifications under the terms of the same license. Which, again, the GSA cannot do, because the very concept of licensing depends on the assertion of copyright.

**Comment by otherjoel** [comment]
@davelab6 

> since its contributions have no copyrights and thus no license, their contributions are 'transparent', and don't effect the underlying/original license situation at all. 

This is a mental model of copyright and licensing which makes internal sense to an OO programmer, but the law is not written that way. If I create any derived work, I own copyright in that work. It enters the world void of any licensing scheme. If I decide to distribute my derivative work, I have to decide what license terms will apply to users of my work, otherwise there is _no_ license. Users of my work don't automatically get the same license as the parent work.

If for some legal reason, such as being the GSA, I cannot hold copyright in my derivative work, the work cannot be placed under the terms of a license. The new work is in the public domain.

The GSA staff involved with this project have made a novel attempt to sidestep the issue by trying to draw a distinction between their modifications and the original font. But they are not distributing these things separately. They are distributing a complete font which their own documentation claims has creative, substantial differences from the original. When you download Public Sans you are not getting, in the eyes of the law, something which is X% copyright Pablo Impallari and Y% public domain. You are getting a font created by the GSA which has no copyright.

**Comment by thlinard** [comment]
https://github.com/uswds/public-sans/blob/develop/LICENSE.md:

> In practice, users of this Modified Version (Public Sans) should use Public Sans according to the terms of the SIL Open Font License, Version 1.1, below. This is because this font is a combination of work subject to copyright and work not subject to copyright, so the more restrictive requirements apply to using the combined work.

**Comment by otherjoel** [comment]
> this font is a combination of work subject to copyright and work not subject to copyright, so the more restrictive requirements apply to using the combined work

As I explained above, this is exactly backwards. The GSA staff have made this distinction up out of whole cloth. The fact that they distribute Public Sans as a complete font (rather than as a patch people could apply to Libre Franklin) renders it meaningless.
